src/z3c/dependencychecker/dependencychecker.py

```diff
@@ -84,6 +84,57 @@
 ['\"]        # Single or double quote.
 """, re.VERBOSE)
 
+FTI_BEHAVIOR_PATTERN = re.compile(r"""
+\s           # Whitespace.
+<element     #
```

Comment:
That's why on the list comprehension it filters any match that does not have a dot in it, we could encode that into the regex of course.

I was just lazy and using the same logic as on the surrounding methods (I basically copy&pasted&adapted ZCML detection code).